pkgs/applications/networking/browsers/firefox-bin/default.nix

```diff
@@ -72,7 +72,7 @@ let
 
   policies = {
     DisableAppUpdate = true;
-  };
+  } // config.firefox.policies or {};
 
   policiesJson = writeText "no-update-firefox-policy.json" (builtins.toJSON { inherit policies; });
```

Comment:
the name is no longer suitable now, should be changed to something like "firefox-policy.json" I think

Reply:
indeed